workflow/layout/letkf_only.yaml

```diff
@@ -55,13 +29,33 @@ suite: !Cycle
         config: [ base ]
         J_JOB: JJOB_GEN_ENSEMBLE
 
+# WORKS for one member
       da_letkf_prep: !Task
         <<: *shared_task_template
         Trigger: !Depend  ( gen_ens_run & obs_prep )
         resources: !calc partition.resources.run_3dvar
         config: [ base ]
         J_JOB: JJOB_PREP_LETKF1
 
+#      da_letkf_prep: !TaskArray
+#        Trigger: !Depend  ( gen_ens_run & obs_prep )
+#        Dimensions:
+#          groupid: !calc tools.seq(1,doc.settings.GROUPS,1)
+#        da_letkf_grp: !TaskElement
+#          <<: *shared_task_template
+#          Foreach: [ groupid ]
+#          Name: !expand "da_letkf_grp{dimval.groupid}"
+#          resources: !calc partition.resources.run_da_prep
+#          config: [ base ]
+#          J_JOB: JJOB_PREP_LETKF
+#          ENSGRP: !expand "{dimval.groupid:02d}"
+#          rocoto_command: !expand >-
+#            {rocoto_load_modules} ;
+#            /usr/bin/env ENSGRP={ENSGRP} &ROOT_GODAS_DIR;/jobs/{J_JOB}
+#          ecflow_command: !expand |
+#            export ENSGRP={ENSGRP}
+#            $ROOT_GODAS_DIR/jobs/{J_JOB}
+
```

Comment:
Do you need to keep these comments?

Reply:
Bug slipped through from the last PR: 
Line 64:      J_JOB: JJOB_DA_LEKTF 
should be 
                  J_JOB: JJOB_DA_**LETKF**